### PR TITLE
docs: Russian translation moved to VuePress

### PR DESCRIPTION
**Pull request by lex111**
cc @Alex-Sokolov

### CODE REVIEW

**Comment by lex111** [comment]
Updated with the latest changes in the docs.

I look forward to @Alex-Sokolov :stuck_out_tongue_winking_eye:

**Comment by lex111** [comment]
Ready for review

**Review by Alex-Sokolov — APPROVED** [review]

**Comment by Alex-Sokolov** [comment]
@lex111 great work! Thanks :)

File: docs/ru/api/createLocalVue.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- **Использование:**
+
+`createLocalVue` возвращает класс Vue, чтобы вы могли добавлять компоненты, примеси и устанавливать плагины без загрузнения глобального класса Vue.
+
+Используйте вместе с `options.localVue`:
+
+```js
+import { createLocalVue, shallow } from '@vue/test-utils'
+import Foo from './Foo.vue'
```
Comment (38elements) [inline docs/ru/api/createLocalVue.md]: shallow -> shallowMount

Comment (lex111) [reply docs/ru/api/createLocalVue.md]: @38elements  thank you! I have not everywhere before made a replacement, now thanks to you, I replaced everywhere with `shallow` on `shallowMount`

**Comment by lex111** [comment]
@eddyerburgh we are a little delayed :disappointed: , but now the documentation in Russian is relevant, look, please.

File: docs/ru/api/config.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+### `mocks`
+
+- type: `Object`
+- default: `{}`
+
+По аналогии с `stubs`, значения, переданные в `config.mocks` используются по умолчанию. Любые значения, переданные настройкам монтирования объекта `mocks`, будут иметь приоритет выше, по сравнению с объявленными в `config.mocks`.
+
+Example:
+
```
Comment (38elements) [inline docs/ru/api/config.md]: Example -> Пример

File: docs/ru/api/config.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```js
+import VueTestUtils from '@vue/test-utils'
+
+VueTestUtils.config.stubs['my-component'] = '<div />'
+```
+
+### `mocks`
+
+- type: `Object`
```
Comment (38elements) [inline docs/ru/api/config.md]: type -> Тип

File: docs/ru/api/config.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- default: `{}`
```
Comment (38elements) [inline docs/ru/api/config.md]: default -> по умолчанию

**Comment by lex111** [comment]
@38elements thanks for the review.

File: docs/ru/api/config.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+- По умолчанию: `{
+  transition: TransitionStub,
+  'transition-group': TransitionGroupStub
+}`
+
+Заглушки указанные в `config.stubs` используются по умолчанию.
+Заглушки, используемые в компонентах. Они перезаписываются значениями `stubs` переданными в настройках монтирования.
+
+При передаче `stubs` в качестве массива в настройках монтирования, `config.stubs` будет преобразована в массив, и будут создаваться компоненты заглушки с базовым компонентом, который возвращает div.
+
```
Comment (38elements) [inline docs/ru/api/config.md]: div -> <${component name}-stub>
https://vue-test-utils.vuejs.org/api/config.html#stubs

File: docs/ru/api/renderToString.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    - `{Object|Array<string>} stubs`
+    - `{Vue} localVue`
+
+- **Возвращает:** `{string}`
+
+- **Опции:**
+
+См. [опции](./options.md)
+
+- **Usage:**
```
Comment (38elements) [inline docs/ru/api/renderToString.md]: Usage -> Использование

**Comment by 38elements** [comment]
#368 is not included.
IMHO, I think this needs to check well.

**Comment by lex111** [comment]
Sorry for the large number of inconsistencies, now I carefully checked everything and corrected the errors. I also updated the Russian translation in accordance with the new changes (_silentWarnings to silent_ and others)

Special thanks to @38elements for reporting discrepancies!

@Alex-Sokolov can you please check again?

Next time we will update this translation together with community Translation Gang, now I acted alone, so it took a long time :face_with_head_bandage:

**Comment by 38elements** [comment]
#493  is not included.
If the document has many incorrect points, many problems and pull requests will be generated.
This increases maintenance cost.

**Comment by lex111** [comment]
@38elements thanks, fixed it.

**Comment by 38elements** [comment]
I took a look at all the files roughly.
I could not find any other problems.
I hope other people review this pull request.

**Comment by 38elements** [comment]
I took a look at all the files roughly.
I could not find the other problems.
I hope other people review this pull request.

File: docs/ru/api/renderToString.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+- **Аргументы:**
+
+  - `{Component} component`
+  - `{Object} options`
+    - `{Object} context`
+      - `{Array<Component|Object>|Component} children`
+    - `{Object} slots`
+        - `{Array<Componet|Object>|Component|String} default`
+        - `{Array<Componet|Object>|Component|String} named`
```
Comment (Alex-Sokolov) [inline docs/ru/api/renderToString.md]: Component — here and line above

File: docs/ru/api/render.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+- **Принимает:**
+
+  - `{Component} component`
+  - `{Object} options`
+    - `{Object} context`
+      - `{Array<Component|Object>|Component} children`
+    - `{Object} slots`
+        - `{Array<Componet|Object>|Component|String} default`
+        - `{Array<Componet|Object>|Component|String} named`
```
Comment (Alex-Sokolov) [inline docs/ru/api/render.md]: Component — here and line above

File: docs/ru/api/renderToString.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+import Foo from './Foo.vue'
+import Bar from './Bar.vue'
+import FooBar from './FooBar.vue'
+
+describe('Foo', () => {
+  it('renders a div', () => {
+    const renderedString = renderToString(Foo, {
+      slots: {
+        default: [Bar, FooBar],
+        fooBar: FooBar, // Will match <slot name="FooBar" />,
```
Comment (Alex-Sokolov) [inline docs/ru/api/renderToString.md]: Not translated yet ("будет соответствовать" in other places)

File: docs/ru/api/render.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+import Foo from './Foo.vue'
+import Bar from './Bar.vue'
+import FooBar from './FooBar.vue'
+
+describe('Foo', () => {
+  it('renders a div', () => {
+    const wrapper = render(Foo, {
+      slots: {
+        default: [Bar, FooBar],
+        fooBar: FooBar, // Will match <slot name="FooBar" />,
```
Comment (Alex-Sokolov) [inline docs/ru/api/render.md]: Not translated yet ("будет соответствовать" in other places)

**Review by Alex-Sokolov — CHANGES_REQUESTED** [review]

**Comment by Alex-Sokolov** [comment]
Also deploy on netlify failed because `7:03:08 PM: failed during stage 'deploying site': Failed to execute deploy: Upload cancelled: zh/guides/using-with-vuex.html`. Maybe need to update some link inside.

File: docs/ru/api/config.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+  }
+}
+```
+
+### `methods`
+
+- Тип: `Object`
+- По умолчанию: `{}`
+
+Вы можете настроить методы по умолчанию с помощью объекта `config`. Это может быть полезно для плагинов, которые вводят методы в компоненты, такие как [VeeValidate](https://vee-validate.logaretm.com/). Вы можете переопределить методы, установленные в `config`, передав `methods` в настройках монтирования.
```
Comment (Alex-Sokolov) [inline docs/ru/api/config.md]: Also need change link to https://baianat.github.io/vee-validate/

**Comment by lex111** [comment]
@Alex-Sokolov thanks you for this, all edits have been made + updated the documentation in accordance with the new changes in original docs.

**Comment by lex111** [comment]
@eddyerburgh please pay attention to this PR :open_hands:

**Review by Alex-Sokolov — APPROVED** [review]

**Review by eddyerburgh — APPROVED** [review]

**Pull request merged**